yandex_geocoder/__init__.py,sha256=NTpmURmCvq750I4Pa1f-GC1bVg9v1FrS-j3BveVDAtE,305
yandex_geocoder/client.py,sha256=k6XRj7jK13yN2K97KnrBl75d0R2Tw0mDn2s-ZSX9Xio,4104
yandex_geocoder/exceptions.py,sha256=yeCZgxfPwz5rH-sczfpr6WmLiU7zA7qKMNfm0GIaudw,334
yandex_geo-2.1.1.dist-info/LICENSE,sha256=G8MXbEVgor2v-R3QOOONb5SHZlQ4oSHWdl7rPA7DlU0,1071
yandex_geo-2.1.1.dist-info/METADATA,sha256=_TeLFBVZQWmvGp92WbJYXocOaY7ir4gpysQCdb_vsbQ,2983
yandex_geo-2.1.1.dist-info/WHEEL,sha256=sP946D7jFCHeNz5Iq4fL4Lu-PrWrFsgfLXbbkciIZwg,88
yandex_geo-2.1.1.dist-info/RECORD,,
